src/components/Tag/Tabs/ImageSelector.jsx

```diff
@@ -0,0 +1,395 @@
+import {
+  Avatar,
+  FormControl,
+  FormHelperText,
+  InputBase,
+  InputLabel,
+  List,
+  ListItem,
+  MenuItem,
+  Select,
+  Stack,
+  Typography
+} from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import PhotoIcon from '@mui/icons-material/Photo';
+import React, { useEffect, useMemo, useState } from 'react';
```

Comment:
while most of our import ordering rules were not necessarily enforced. We should try to always keep the React import as the first one